source/eitri/ed_handler.py

```diff
@@ -30,7 +30,6 @@
 import glob
 import os
 from navitiacommon import utils, launch_exec
-from navitiacommon.launch_exec import launch_exec
```

Comment:
Are you sure that eitri still works after that?

Reply:
You're right I have this error :
'module' object is not callable

I'm investigating ...

Reply:
Fix done.